### DISCUSSION TITLE
How to access Grafana through API calls?

### DISCUSSION THREAD

**Discussion by nasru1994** [Security-Onion-Solutions/securityonion#7723] [Unsupported Versions] [answered]
I have my Grafana up & running I can access it through my web browser. But I want to access it through api calls. For example I want to list the orgs & dashboards, in default Grafana (without security onion) i can access them using the following urls 
1. https://{grafana-baseurl}/api/user/orgs/
2. https://{grafana-baseurl}/api/datasources/
I authenticated the above urls by using API keys provided by Grafana.

When I tried to access the same urls for a security onion hosted Grafana I get the security onion login page html as a response. For ex: https://{sos-baseurl}/grafana/api/user/orgs/

Do i have to expose the Grafana API somewhere or Is there anyway to achieve my use case?

**Comment by dougburks** [comment, marked as answer]
In Security Onion, Grafana is protected via our Security Onion login page. We don't support changing this.
